### ISSUE TITLE
Add support for scoverage

### ISSUE THREAD

**Issue by MasseGuillaume** [scala-native/scala-native#627]
scoverage cannot be enabled with scala-native:

```
[info] [info] Will write measurement data to [/home/gui/scala-native/unit-tests/target/scala-2.11/scoverage-data]
scala.scalanative.util.package$UnsupportedException: can't recover runtime tag from {
  scoverage.Invoker.invoked(6141, "/home/gui/scala-native/unit-tests/target/scala-2.11/scoverage-data");
  native.this.Tag.Byte()
} (class java.lang.String)
        at scala.scalanative.util.package$.unsupported(package.scala:11)
        at scala.scalanative.nscplugin.NirCodeGen$NirCodePhase$$anonfun$unwrapTag$1.apply(NirCodeGen.scala:1495)
```

**Comment by densh** [comment]
Related: #396

**Comment by densh** [comment]
Fixed in #1426
